client/package.json

```diff
@@ -22,7 +22,7 @@
     "react-dom": "^17.0.2",
     "react-github-login": "^1.0.3",
     "react-hot-toast": "^1.0.2",
-    "react-icons": "^4.2.0",
+    "react-icons": "4.2.0",
```

Comment:
react-icons v4.3.0 is broken and was causing npm run build and start to fail, and the fix is to force the older version, see https://github.com/react-icons/react-icons/issues/490#issuecomment-932823191 
also checked that the package-lock.json is using v4.2.0

Reply:
Nice catch. Luckily it seems they just fixed it yesterday with 4.3.1:
https://github.com/react-icons/react-icons/issues/490#issuecomment-932996600

So I think we can just leave it at ^4.2.0.